superset-frontend/src/explore/components/ExploreChartPanel.jsx

```diff
@@ -324,7 +324,9 @@ const ExploreChartPanel = ({
                 <span
                   role="button"
                   tabIndex={0}
-                  onClick={() => setShowDatasetModal(true)}
+                  onClick={() => {
```

Comment:
Ditto above about this being a virtual dataset creation action.

Reply:
Confirmed: virtual only

Reply:
Fixed in [`this commit`](https://github.com/apache/superset/pull/22610/commits/d2903e307228270c40270088b7a83d64ae13e545).